Databases: surface differences between distributions of Postgres (#6475)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/databases/postgres_distributions/index.mdx

deepToC: true
navigation: 
  - feature_support_per_db
  - advanced_replication
  - advanced_security
  - advanced_sql
  - advanced_performance
  - oracle_compatibility
  - tools_extensions
---

EDB builds upon the upstream open-source [PostgreSQL](https://www.postgresql.org/) project, enhancing it with additional capabilities designed to support a wider range of use cases and enterprise requirements.

<br/>

![Postgres distributions comparison by features](/edb-postgres-ai/databases/images/postgres_distros_transparent_500w.svg)

## Postgres distributions

### PostgreSQL

PostgreSQL offers all the features gathered in the [PostgreSQL documentation](https://www.postgresql.org/docs/current/).

You can obtain PostgreSQL from our [EDB repository](https://www.enterprisedb.com/software-downloads-postgres), for which you require an EDB account and license. Alternatively, you can download it from the [official PostgreSQL site](https://www.postgresql.org/download/).

### EDB Postgres Extended Server 

Additionally to all the features PostgreSQL offers, EDB Postgres Extended Server also adds a number of features to support [transparent data encryption](/tde/latest).

> When combined with our powerful high-availability and multi-master replication solution, [EDB Postgres Distributed](/pgd/latest), advanced replication is also available for EDB Postgres Extended Server.

Reviewer: replication... replication?

```suggestion
EDB Postgres Extended Server can also be combined with and fully utilise EDB's powerful high-availability and multi-master replication solution, [EDB Postgres Distributed](/pgd/latest).
```

You can obtain EDB Postgres Extended Server from our [EDB repository](https://www.enterprisedb.com/software-downloads-postgres), for which you require an EDB account and license.

### EDB Postgres Advanced Server 

> Additionally to all the features included in PostgreSQL and EDB Postgres Extended Server, EDB Postgres Advanced Server also adds a number of features to support [advanced security](/epas/latest/epas_security_guide), [advanced SQL](/epas/latest/application_programming/15_enhanced_sql_and_other_misc_features/) and, if required, [Oracle compatibility](/epas/latest/tools_utilities_and_components).

Reviewer: ```suggestion
EDB Postgres Advanced Server builds on the features of EDB Postgres Extended Server and supports features such as [advanced security](/epas/latest/epas_security_guide), [advanced SQL](/epas/latest/application_programming/15_enhanced_sql_and_other_misc_features/) and, optional [Oracle compatibility](http://enterprisedb.com/docs/epas/latest/working_with_oracle_data/).
```

Oracle link was to tools, not to the actual compatibility section.
